manual/manual/Makefile

```diff
@@ -39,25 +39,35 @@ index:
 	  && makeindex manual.kwd.idx
 
 
-# libref/style.css is used as witness for the generation of the html stdlib
-# reference.
-html: htmlman/libref/style.css etex-files
+# libref/style.css and comilerlibref/style.css are used as witness
+# for the generation of the html stdlib and compilerlibs reference.
+html: htmlman/libref/style.css htmlman/compilerlibref/style.css etex-files
 	cd htmlman \
 	  && $(HEVEA) $(HTML_FLAGS) \
 	    -I .. -I ../cmds -I ../library -I ../refman -I ../tutorials \
 	    -I ../../styles -I ../texstuff \
 	    manual.hva -e macros.tex ../manual.tex \
 	  && $(HACHA) -tocter manual.html
 
-htmlman/libref/style.css: style.css $(STDLIB_DEPS)
+htmlman/libref/style.css: style.css $(STDLIB_MLIS)
 	mkdir -p htmlman/libref
 	$(OCAMLDOC) -colorize-code -sort -html \
 	  -d htmlman/libref \
-	  $(DOC_INCLUDES) \
+	  $(DOC_STDLIB_INCLUDES) \
 	  $(STDLIB_MLIS)
 	cp style.css $@
 
 
+htmlman/compilerlibref/style.css: style.css $(COMPILERLIBS_MLIS)
+	mkdir -p htmlman/compilerlibref
+	$(OCAMLDOC) -colorize-code -sort -html \
+	  -d htmlman/compilerlibref \
+	  -I $(SRC)/stdlib \
```

Comment:
Why is the line above `-I ../stdlib`, instead of `$(DOC_STDLIB_INCLUDES)` that also includes otherlibs modules like Str and Unix? Are there no references from compiler-libs to otherlibs?

Reply:
This stdlib include is here to typecheck the compiler-libs modules, which have no dependencies on otherlibs.